include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -568,7 +568,7 @@ struct __parallel_copy_if_static_single_group_submitter<_Size, _ElemsPerItem, _W
 
         constexpr ::std::uint32_t __elems_per_wg = _ElemsPerItem * _WGSize;
         using __result_and_scratch_storage_t = __result_and_scratch_storage<_Policy, _Size>;
-        __result_and_scratch_storage_t __result{__policy, 0};
+        __result_and_scratch_storage_t __result{__policy, 1, 0};
```

Comment:
If we apply https://github.com/oneapi-src/oneDPL/pull/1773/files#r1711563149,
we able to rewrite this as `__result_and_scratch_storage_t __result{__policy, 1};`

Reply:
See above.